openwisp_network_topology/migrations/0001_initial.py

```diff
@@ -61,7 +63,7 @@ class Migration(migrations.Migration):
                 ('parser', models.CharField(choices=[('netdiff.OlsrParser', 'OLSRd (txtinfo/jsoninfo)'), ('netdiff.BatmanParser', 'batman-advanced (jsondoc/txtinfo)'), ('netdiff.BmxParser', 'BMX6 (q6m)'), ('netdiff.NetJsonParser', 'NetJSON NetworkGraph'), ('netdiff.CnmlParser', 'CNML 1.0')], help_text='Select topology format', max_length=128, verbose_name='format')),
                 ('strategy', models.CharField(choices=[('fetch', 'FETCH'), ('receive', 'RECEIVE')], db_index=True, default='fetch', max_length=16, verbose_name='strategy')),
                 ('url', models.URLField(blank=True, help_text='Topology data will be fetched from this URL (FETCH strategy)', verbose_name='url')),
-                ('key', models.CharField(blank=True, default=django_netjsongraph.utils.get_random_key, help_text='key needed to update topology from nodes ', max_length=64, verbose_name='key')),
+                ('key', models.CharField(blank=True, default=get_random_key, help_text='key needed to update topology from nodes ', max_length=64, verbose_name='key')),
```

Comment:
in order to fix the build you need to change this line in a similar way to how it was changed in django-netjsongraph, see https://github.com/openwisp/django-netjsongraph/commit/110cb7175526bfe105d5a6a16f58085dcf6a5103#diff-8d08650f6724971ea8f07469294f2f33L27

Reply:
:+1: